docs/Parallel-Learning-Guide.rst

```diff
@@ -121,9 +121,9 @@ While setting up for training, ``lightgbm`` will concatenate all of the partitio
 
 When setting up data partitioning for LightGBM training with Dask, try to follow these suggestions:
 
-* ensure that each worker in the cluster has some of the training data
-* try to give each worker roughly the same amount of data, especially if your dataset is small
-* if you plan to train multiple models (for example, to tune hyperparameters) on the same data, use ``client.persist()`` before training to materialize the data one time
+* ensure that each worker in the cluster has some of the training data;
```

Comment:
Why is it necessary to add all these semicolons?

Reply:
Sorry, as you know I'm not a native English speaker, I thought these semicolons were just accidentally missed.

> Semicolons were traditionally used in lists to separate each bullet point, but although this is still correct, they are not used as often today. 
   https://www.onlinegrammar.com.au/punctuation-in-lists/

Now I see that I was wrong. But it looks like we need a full stop at the end at least, right?

> A common style is to use lower case for the first letter in each bullet point, and a full stop after the last bullet point.

Reply:
No problem, totally understood!

I don't think you need to worry about this. For native speakers (in American English, at least...that website you linked to is from Australia 😁 ) bulleted lists don't really have formal rules.  Any mix of the following is equally valid:

* capitalizing vs. not capitalizing the first letter of each bullet point
* using a period at the end of each bullet point
* using semicolons at the end of the first `n-1` bullet points and a period at the end of the final one

So it's not that the current state on `master` is "right" and your proposal is "wrong". They're all equally valid ways to write a bulleted list, and I don't think it's worth spending time worrying about it.

Reply:
Got it! Thank you very much for the lesson! 🙂